heroku/cli
# refactor(container-registry-v5): move container:pull to CLI & upgrade to oclif

### Description

This converts `container:pull` to oclif and updates its associated test.

### Testing

1) Create an app based on a Dockerfile
2) Run `./bin/run container:pull web -a APP_NAME`
3) Verify the command succeeds and the output is correct
3) Compare the results and make sure things look 👍
